pkg/controller/controllername/controllername.go

```diff
@@ -28,12 +28,12 @@ type ControllerNamer interface {
 	AddToController(blder *builder.Builder, l logr.Logger) *builder.Builder
 }
 
-// controllerName ex. {"My","Controller", "Name"} ->  MyControllerName
-type controllerName []string
+// ControllerName ex. {"My","Controller", "Name"} ->  MyControllerName
```

Comment:
why are we exporting this?

Reply:
having it private was purposeful to force them to use New making it difficult to misues

Reply:
It helps keep our tests concise - check out the changes I made in the placeholder pod tests.

how would you feel about making it a separate struct containing a private string list? That would allow us to achieve the same

Reply:
actually wait I think I can work around this

Reply:
ok reverted to private with a bit of a workaround